src/structure_managers/species_manager.hh

```diff
@@ -30,34 +28,91 @@
  * Boston, MA 02111-1307, USA.
  */
 
-/*
- * ==================================================
- *  OBSOLETE NOT NECESSARY TO CHANGE, EOL DETERMINED
- * ==================================================
- */
-
 #ifndef SPECIES_MANAGER_H
 #define SPECIES_MANAGER_H
 
 #include "structure_managers/structure_manager.hh"
-#include "representations/basis_function_manager.hh"
+#include "structure_managers/adaptor_filter.hh"
+#include "structure_managers/property.hh"
+
+#include <type_traits>
+#include <array>
+#include <tuple>
+#include <map>
+#include <memory>
 
 namespace rascal {
 
-  template <class NeighManager, int MaxLayer, int Layer = 0>
+  namespace internal {
+
+    namespace {
```

Comment:
Any specific reason we need an unnamed namespace here? This practice is discouraged for header files.

Reply:
Nevermind, just read the change to the coding convention. I guess it makes sense for implementation-heavy header files, but didn't we agree to be moving this sort of thing to separate "implementation header" (`_impl.hh`) files?